trendy-fashion/src/Components/Customer.jsx

```diff
@@ -1,11 +0,0 @@
-import React from 'react'
-import { Image } from '@chakra-ui/react'
-const Customer = () => {
-    return (
-        <div>
```

Comment:
Instead of empty <div> tags, prefer to use React fragments, because <div> tags takes longer to render on browser